internal_deps.bzl

```diff
@@ -10,6 +10,14 @@ load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
 def http_archive(name, **kwargs):
     maybe(_http_archive, name = name, **kwargs)
 
+def rules_py_rust_deps():
```

Comment:
nit: perhaps this should be `rules_py_tool_deps`? The implementation language is just that, an implementation detail that doesn't need exposing here, and if we add others, this doesn't need to change.

Reply:
sure - I wish rules_rust would change so we could tuck the dependency in here altogether, right now the rust is quite leaky since it's a three-time load/init dance in WORKSPACE

Reply:
looking at this again, I think it's not even worth having this in its own function. WORKSPACE users on a prerelease will still have to load/call all of these:

```
rules_rust_dependencies()
rust_register_toolchains()
crate_universe_dependencies()
crates_repository()
```

may as well force them to have the trivial `http_archive` for rules_rust too